python/lsst/pipe/tasks/postprocess.py

```diff
@@ -1976,6 +1976,50 @@ def run(self, inputCatalog, inputCatalogDiff, ccdVisitId=None, band=None):
         return pipeBase.Struct(outputCatalog=outputCatalog)
 
 
+class RewriteForcedSourceOnDiaObjectConnections(WriteForcedSourceTableConnections):
+    exposure = connectionTypes.Input(
+        doc="Input exposure to perform photometry on.",
+        name="calexp",
+        storageClass="ExposureF",
+        dimensions=["instrument", "visit", "detector"],
+    )
+
+
+class RewriteForcedSourceOnDiaObjectConfig(WriteForcedSourceTableConfig,
+                                           pipelineConnections=RewriteForcedSourceOnDiaObjectConnections):
+    reEvaluate = pexConfig.ConfigurableField(
+        target=WriteRecalibratedSourceTableTask,
+        doc="Subtask with addCalibColumns method",
+    )
+
+
+class RewriteForcedSourceOnDiaObjectTask(WriteForcedSourceTableTask):
+    """Specialized afterburner to recalibrate DP0.2 ForcedSourceOnDiaObject"""
+    _DefaultName = "RewriteForcedSourceOnDiaObject"
+    ConfigClass = RewriteForcedSourceOnDiaObjectConfig
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.makeSubtask("reEvaluate")
+
+    def runQuantum(self, butlerQC, inputRefs, outputRefs):
+        inputs = butlerQC.get(inputRefs)
+        # Add ccdVisitId to allow joining with CcdVisitTable
+        inputs['ccdVisitId'] = butlerQC.quantum.dataId.pack("visit_detector")
+        inputs['band'] = butlerQC.quantum.dataId.full['band']
+        inputs['exposureIdInfo'] = ExposureIdInfo.fromDataId(butlerQC.quantum.dataId, "visit_detector")
+
+        inputs['inputCatalog'] = self.reEvaluate.addCalibColumns(catalog=inputs['inputCatalog'],
+                                                                 exposure=inputs['exposure'],
+                                                                 exposureIdInfo=inputs['exposureIdInfo'])
+        inputs['inputCatalogDiff'] = self.reEvaluate.addCalibColumns(catalog=inputs['inputCatalogDiff'],
+                                                                     exposure=inputs['exposure'],
+                                                                     exposureIdInfo=inputs['exposureIdInfo'])
+
+        outputs = self.run(**inputs)
```

Comment:
When I ran this, it complained that `run()` doesn't accept a kwarg `exposure`. Are you missing a git push?

Reply:
Oh my. Yes, I did miss a push!